records_mover/db/redshift/redshift_db_driver.py

```diff
@@ -66,7 +66,7 @@ def table(self, schema: str, table: str) -> Table:
                 # There was some hang-wringing when they added it originally:
                 #  https://github.com/sqlalchemy-redshift/sqlalchemy-redshift/commit/
                 #      94723ec6437c5e5197fcf785845499e81640b167
-                return Table(table, self.meta, schema=schema, autoload=True, autoload_with=conn)
+                return Table(table, self.meta, schema=schema, autoload_with=conn)
```

Comment:
wait, now im confused. Earlier we passed `autoload_with=self.db_engine`, but now we are passing the connection? i thought using the engine/connection as a union was gone in 2.0?